src/watts_ui/etc/watts.sch

```diff
@@ -0,0 +1,370 @@
+ChildExactlyOne = [ watts ]
+watts{
+    Description = "[optional] for watts calculations"
+    MinOccurs = 0
+    MaxOccurs = 1
+    InputTmpl = "watts"
+    workflow_level1{
+        Description = "[required] Workflow definition - first level"
+        InputTmpl="sonobject"
```

Comment:
instead of "sonobject" - let's replace with `workflow_level1` and develop one `workflow_level1.tmpl` that shows example of workflow options with a `plugin`, a `variables` block, and commented `iteration` and `parametric` blocks

Reply:
Developed templates for plugin, variables, etc. Workbench can now auto generate these blocks.